idb/postgres/postgres_migrations.go

```diff
@@ -821,6 +821,28 @@ func m7RewardsAndDatesPart2UpdateAccounts(db *IndexerDb, accountData []addressAc
 	return nil
 }
 
+func warnUser(db *IndexerDb, maxRound uint32) error {
+	query := "SELECT COUNT(*) FROM account WHERE (created_at IS NULL) OR (created_at <= $1)"
+	row := db.db.QueryRow(query, maxRound)
+
+	var count uint64
+	err := row.Scan(&count)
+	if err != nil {
+		return fmt.Errorf("m7: unable to query the number of rows: %v", err)
+	}
+
+	if count > 10000000/3*4 {
+		db.log.Print("the migration m7 is likely to use more than 4GB of RAM")
+
+		envVar := "FORCEM7"
+		if _, ok := os.LookupEnv(envVar); !ok {
+			return fmt.Errorf("m7: set %s environment variable to force the migration", envVar)
```

Comment:
It comes right after so - "To force start the migration, please set the environment variable %s to TRUE". I know that the work doesn't matter but it's better to be explicit with the user.